### ISSUE TITLE
Accessibility for 'New Visit' form

### ISSUE THREAD

**Issue by dc2007git** [rcpch/national-paediatric-diabetes-audit#49]
The 'Add new visit' patient form looks great and very much like it fits into a clinical setting. My only thought is that for those with colourblindness, it might be more difficult to see, and thus we should decide on potentially a 'plainer' colourscheme

**Comment by eatyourpeas** [comment]
Absolutely. The colours were only introduced to highlight the concept of categories. I wonder if we should instead have a series of tabs, or a menu bar to navigate to each category, with the more commonly used ones at the top, such as HbA1c, weight/height/BP/CGM and treatment.

Also, would be good to prepopulate the date, for example of visit date to today so the user only has to accept, rather than open a date picker. I am starting to feel that htmx might be good for this form.

**Comment by mbarton** [comment]
The form now only uses colour for the category description.
